### ISSUE TITLE
MessageInput property inputCursorDrawable does not work.

### ISSUE THREAD

**Issue by ryanthon** [stfalcon-studio/ChatKit#23]
I am trying to set a custom cursor drawable for the MessageInput EditText, but it is not working. I am using a simple rectangle shape, and I have tested this in other EditTexts, and it works:

```
<?xml version="1.0" encoding="utf-8"?>
<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle" >
    <size android:width="1dp" />
    <solid android:color="@android:color/black" />
</shape>
```

The property I'm setting in XML is:

`app:inputCursorDrawable="@drawable/black_cursor"`

**Comment by troy379** [comment]
Hello @ryanthon 

I have no idea, how could I miss this. Now it's fixed in version `0.2.1`.
Thanks!

Best regards, Alexander

**Comment by SteveByrne91** [comment]
I'm having the same issue on 0.3.3.
Is it possible this bug was reintroduced?
